src/main/resources/explanations/challenge29_hint.adoc

```diff
@@ -0,0 +1,10 @@
+
+So we are looking for a secret that only you can see!
```

Comment:
Please have a look at the other hint-texts. the idea is that we guide the user step by step on what to do.